src/transformStateWithClones.js

```diff
@@ -7,7 +7,35 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  const arrOfObjects = [];
+  const copyState = Object.assign({}, state);
+
+  for (const obj of actions) {
+    switch (obj.type) {
+      case 'addProperties':
+        Object.assign(copyState, obj.extraData);
+        break;
+      case 'removeProperties':
```

Comment:
add line breaks between cases, hard to read
```suggestion
        break;

      case 'removeProperties':
```